Dockerfile.venv

```diff
@@ -10,4 +13,4 @@ RUN virtualenv venv
 ENV VIRTUAL_ENV=${PROJECT_FOLDER}/venv
 
 COPY Pipfile Pipfile.lock ./
-RUN pipenv install ${pipenv_dev_arg} --deploy
+RUN pipenv sync ${pipenv_dev_arg}
```

Comment:
Actually it's documented here: https://pipenv.readthedocs.io/en/latest/advanced/

> pipenv install --ignore-pipfile is nearly equivalent to pipenv sync, but pipenv sync will never attempt to re-lock your dependencies as it is considered an atomic operation. pipenv install by default does attempt to re-lock unless using the --deploy flag.

(The docs for `--ignore-pipfile` should really point to that)

Reply:
Yes, `sync` seemed marginally better so I went for that